gobblin-utility/src/main/java/gobblin/broker/iface/KeyedConfigView.java

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright (C) 2014-2016 LinkedIn Corp. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use
+ * this file except in compliance with the License. You may obtain a copy of the
+ * License at  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed
+ * under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+ * CONDITIONS OF ANY KIND, either express or implied.
+ */
+
+package gobblin.broker.iface;
+
+import com.typesafe.config.Config;
+
+
+/**
+ * Provides a view of a {@link Config} specific to a {@link SharedResourceKey} and {@link SharedResourceFactory}.
+ */
+public interface KeyedConfigView<K extends SharedResourceKey> extends ConfigView {
```

Comment:
What is the interface for? What are other use cases besides `KeyedScopedConfigView`?